Description: Drop code where neutron drops privileges from root.
  This code isn't required because in strict mode we run everything
  as root:root and setuid is not allowed by the current plugs.
Author: Corey Bryant <corey.bryant@canonical.com>
Forwarded: no

---
 neutron/agent/linux/daemon.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/neutron/agent/linux/daemon.py b/neutron/agent/linux/daemon.py
index 6ad9ee3c8..23218eace 100644
--- a/neutron/agent/linux/daemon.py
+++ b/neutron/agent/linux/daemon.py
@@ -255,4 +255,3 @@ class Daemon(object):
         """
         if not self.watch_log:
             unwatch_log()
-        drop_privileges(self.user, self.group)
-- 
2.11.0

